package.json

```diff
@@ -23,7 +23,8 @@
   "authors": [
     "Dzmitry_Shaplyka@epam.com",
     "Oleksandr_Halichenko@epam.com",
-    "John_Doe@epam.com"
+    "John_Doe@epam.com",
+    "Hleb Iuko"
```

Comment:
Please put `Hleb_Iuko@epam.com`
(a little bit up you have the example's format for name)